data/brands/leisure/trampoline_park.json

```
@@ -52,6 +52,17 @@
         "leisure": "trampoline_park",
         "name": "Sky Zone"
       }
+    },
+    {
+      "displayName": "Urban Air",
+      "id": "urbanair-02op54",
+      "locationSet": {"include": ["us"]},
```

They also have a location in Vienna
```suggestion
      "locationSet": {"include": ["at", "us"]},
```